static/app/views/insights/common/utils/convertSeriesToTimeseries.tsx

```diff
@@ -1,8 +1,8 @@
-import type {TimeseriesData} from 'sentry/views/dashboards/widgets/common/types';
```

Comment:
I am of a strong belief that generic words like 'Data' should just never exist in any naming

Reply:
I agree! The temptation is _so_ strong there. Words like "response", "data", etc. come up all the time. For time series, we've found better more specific alternatives for _what_ the data is (e.g., `EventsStats`, `GroupedEventsStats`, and so on) but it requires a lot of discipline